catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java

```diff
@@ -553,12 +554,14 @@ public Response getDocument(@Encoded @PathParam("sourceid") String encodedSource
                         "Specified query is unsupported.  Change query and resubmit: ";
                 LOGGER.warn(exceptionMessage, e);
                 throw new ServerErrorException(exceptionMessage, Status.BAD_REQUEST);
-                // The catalog framework will throw this if any of the transformers blow up. We need to
-
-                // catch this exception
-                // here or else execution will return to CXF and we'll lose this message and end up with
-                // a huge stack trace
-                // in a GUI or whatever else is connected to this endpoint
+            } catch (DataUsageLimitExceededException e) {
+                String exceptionMessage = "Unable to process request. Data usage limit exceeded: ";
+                LOGGER.warn(exceptionMessage, e);
```

Comment:
Since this is an "expected" error, should we log at DEBUG level instead?

If we do, we should make the same change on master as well.